packages/gatsby-cli/src/reporter/index.js

```diff
@@ -179,6 +179,11 @@ const reporter: Reporter = {
       span,
     }
   },
+  _setStage(stage) {
```

Comment:
why is this method with underscore?

Reply:
It's a convention to denote that this method is private and should not be used outside of Gatsby Core.

Reply:
thx

Reply:
yeah, the implementation will change in the near future so I don't want anyone to start using it in the meantime.